charts/ccm-hetzner/values.yaml

```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 
 image:
-  repository: quay.io/syself/hetzner-cloud-controller-manager
+  repository: ghcr.io/syself/hetzner-cloud-controller-manager:v1.18.0-0.0.4
```

Comment:
please remove the tag here. The tag is already set with appVersion

Reply:
fixed.